src/test/java/george/lost/to/a/bot/StoopidProject/addNumsTest.java

```diff
@@ -0,0 +1,19 @@
+package george.lost.to.a.bot.StoopidProject;
```

Comment:
Williamson is a loli lover

Reply:
Shouldn't we name this MathUtilsTest?

Reply:
renamed the file and added the commit to the pull request

Reply:
George will be the grunt.